packages/trip-form/src/__mocks__/modes-fr.ts

```diff
@@ -0,0 +1,193 @@
+import bikeImage from "./static/bike.svg";
+import carImage from "./static/car.svg";
+import scooterImage from "./static/scooter.svg";
+
+const commonModes = {
+  transitModes: [
+    {
+      image: "https://trimet.org/global/img/mode-bus.png",
+      mode: "BUS",
+      label: "Bus"
+    },
+    {
+      image: "https://trimet.org/global/img/mode-max.png",
+      mode: "TRAM",
+      label: "MAX & Portland Streetcar"
+    },
+    {
+      image: "https://trimet.org/global/img/mode-wes.png",
+      mode: "RAIL",
+      label: "WES Trains régionaux"
```

Comment:
Here again would it not maybe make more sense to store internationalization keys?